diffs/preAaveV3_Arb_ARBListing_20231207_postAaveV3_Arb_ARBListing_20231207.md

```diff
@@ -0,0 +1,124 @@
+## Reserve changes
+
+### Reserves added
+
+#### ARB ([0x912CE59144191C1204E64559FE8253a0e49E6548](https://arbiscan.io/address/0x912CE59144191C1204E64559FE8253a0e49E6548))
+
+| description | value |
+| --- | --- |
+| decimals | 18 |
+| isActive | true |
+| isFrozen | false |
+| supplyCap | 20,000,000 ARB |
+| borrowCap | 16,000,000 ARB |
```

Comment:
Can you also update the diffs as they have changed :)

Reply:
fixed